src/calculator.ts

```diff
@@ -1,54 +1,86 @@
 import fs from "fs";
 import { linearQF, Contribution, Calculation } from "pluralistic";
+import {AugmentedResult, CalculatorOptions, RawContribution} from "./types.js";
 
-type AugmentedResult = Calculation & {
-  projectName: string;
-  payoutAddress: string;
-};
-
-interface RawContribution {
-  voter: string;
-  projectId: string;
-  amountUSD: number;
-}
 
 export default class Calculator {
   private baseDataPath: string;
   private chainId: string;
   private roundId: string;
   private matchAmount: number;
+  private minimumAmount: number | undefined;
+  private passport: boolean | undefined;
```

Comment:
Could we have just a `passportThreshold?: number` here to accomplish the same thing, isn't the boolean redundant?

Reply:
I thought about this - and I think it could be necessary.

For example, if a user wanted to use passport but with the set values. 
The `passportThreshold` is meant to overwrite the default value, if any. 

e.g. 
To use passport with the configured passport threshold, they set `?passport=true`. 
If they want to overwrite the threshold - then they use `?passport=true&passportThreshold=x`


I suppose we could narrow this down to a singular variable - and just parse whether its a string or number, such as `?passportThreshold=x | ?passportThreshold=default`

Reply:
that makes sense!